d2cli/main.go

```diff
@@ -114,6 +114,7 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	boardFlag := ms.Opts.String("D2_BOARD", "board", "", "", "render a single board from the diagram. E.g. root.scenarios.name to render scenario 'name'. If not provided, the entire diagram is rendered.")
```

Comment:
Could you update the man page with this new flag too?

https://github.com/terrastruct/d2/blob/master/ci/release/template/man/d2.1

Reply:
Sure, I also added this feature to changelog.